pkgs/development/tools/github-changelog-generator/default.nix

```
@@ -0,0 +1,14 @@
+{ stdenv, lib, bundlerApp}:
+bundlerApp rec {
+  pname = "github_changelog_generator";
+  gemdir = ./.;
+  exes = ["github_changelog_generator"];
```

We usually like spaces inside lists:

```suggestion
  exes = [ "github_changelog_generator" ];
```